### PR TITLE
feat(layout): changed avatar location

### PR DESCRIPTION
**Pull request by america-nishi**
moved avatar inside the drawer

### CODE REVIEW

File: src/features/channels/components/chat-header/ChatHeader.styles.ts [hunk 1]
```diff
@@ -22,16 +22,16 @@ const useStyles = makeStyles(() =>
...
-      width: '10%',
-      display: 'flex',
-      justifyContent: 'center',
-      alignItems: 'center',
-    },
-    accountAvatar: {
-      width: '4rem',
-      height: '4rem',
-    },
+    // account: {
```
Comment (mmiranda-stackit) [inline src/features/channels/components/chat-header/ChatHeader.styles.ts:25]: I suggest to not let commented code

File: src/features/channels/components/chat-header/ChatHeader.tsx [hunk 2]
```diff
@@ -55,9 +54,9 @@ const ChatHeader = () => {
             <MoreHorizIcon />
           </IconButton>
         </Box>
-        <Box className={classes.account}>
+        {/* <Box className={classes.account}>
```
Comment (mmiranda-stackit) [inline src/features/channels/components/chat-header/ChatHeader.tsx:57]: Same than above, please remove commented blocks of code

**Review by mmiranda-stackit — CHANGES_REQUESTED** [review]

File: src/features/channels/components/chat-header/ChatHeader.styles.ts (hunk 1 above)
Comment (america-nishi) [reply src/features/channels/components/chat-header/ChatHeader.styles.ts:25]: ok

File: src/features/channels/components/chat-header/ChatHeader.tsx (hunk 2 above)
Comment (america-nishi) [reply src/features/channels/components/chat-header/ChatHeader.tsx:57]: ok, I will remove the comments

**Pull request merged**